### DISCUSSION TITLE
Where did Blame go?

### DISCUSSION THREAD

**Discussion by mcsilver** [gitextensions/gitextensions#12103] [Q&A] [answered]
Not sure when it changed, but I now cannot find the Blame tab when I show a file's history. Using Git Bash, I can do a `git blame`, but it would be nice to use Git Extensions.

Git Extensions x64 5.1.1.17970
Git for Windows 2.47.1

According to this:

https://git-extensions-documentation.readthedocs.io/en/release-5.1/file_history.html#blame

the File History window should have the Blame tab; I noticed that my window's title is no longer File History. Maybe that's a clue?

Below is the window I get when I right click on a file, Git Extensions > File history:

![Git Extensions Missing Blame Tab](https://github.com/user-attachments/assets/aa01f5ba-ebc4-4b7c-9a52-eed81265d20d)

**Comment by pmiossec** [comment, marked as answer]
It seems that you don't have a revision selected (so nothing to blame...).

After that, press 'B' on a file in filetree or diff tab.

**Reply by mcsilver** [reply]
Thanks. For some reason, the blame info was toggle off, and typing B toggled it back on.
